eos/effects/industrialcoreeffect2.py

```diff
@@ -1,29 +1,100 @@
-                                     module.getModifiedItemAttr("shipBonusORECapital3"),
-                                     skill="Capital Industrial Ships",
-                                     )
+    fit.ship.boostItemAttr("maxVelocity", src.getModifiedItemAttr("speedFactor"))
+    fit.ship.multiplyItemAttr("mass", src.getModifiedItemAttr("siegeMassMultiplier"))
+    fit.ship.multiplyItemAttr("scanResolution", src.getModifiedItemAttr("scanResolutionMultiplier"))
+
+    #  Remote Shield Repper Bonuses
+    fit.modules.filteredItemBoost(lambda mod: mod.item.requiresSkill("Capital Shield Emission Systems"),
+                                  "duration",
+                                  src.getModifiedItemAttr("industrialCoreRemoteLogisticsDurationBonus"),
+                                  )
+    fit.modules.filteredItemBoost(lambda mod: mod.item.requiresSkill("Capital Shield Emission Systems"),
+                                  "capacitorNeed",
+                                  src.getModifiedItemAttr("industrialCoreRemoteLogisticsRangeBonus"),
+                                  )
+
+    #  Local Shield Repper Bonuses
+    fit.modules.filteredItemBoost(lambda mod: mod.item.requiresSkill("Capital Shield Operation"),
+                                  "duration",
+                                  src.getModifiedItemAttr("industrialCoreLocalLogisticsDurationBonus"),
+                                  )
+    fit.modules.filteredItemBoost(lambda mod: mod.item.requiresSkill("Capital Shield Operation"),
+                                  "shieldBonus",
+                                  src.getModifiedItemAttr("industrialCoreLocalLogisticsAmountBonus"),
+                                  )
+
+    # Mining Burst Bonuses
+    fit.modules.filteredItemBoost(lambda mod: mod.item.requiresSkill("Mining Foreman"),
+                                  "warfareBuff1Value",
+                                  src.getModifiedItemAttr("industrialCoreBonusMiningBurstStrength"),
+                                  )
+
+    fit.modules.filteredItemBoost(lambda mod: mod.item.requiresSkill("Mining Foreman"),
+                                  "warfareBuff2Value",
+                                  src.getModifiedItemAttr("industrialCoreBonusMiningBurstStrength"),
+                                  )
+
+    fit.modules.filteredItemBoost(lambda mod: mod.item.requiresSkill("Mining Foreman"),
+                                  "warfareBuff3Value",
+                                  src.getModifiedItemAttr("industrialCoreBonusMiningBurstStrength"),
+                                  )
+
+    fit.modules.filteredItemBoost(lambda mod: mod.item.requiresSkill("Mining Foreman"),
+                                  "warfareBuff4Value",
+                                  src.getModifiedItemAttr("industrialCoreBonusMiningBurstStrength"),
+                                  )
+
+    #  Command Burst Range Bonus
+    fit.modules.filteredItemBoost(lambda mod: mod.item.group.name in
```

Comment:
Other items require the Leadership skill (fighters, for example).

Why not filter on just the group? Lots of other effects do so.

Alternatively we can filter on skills and target the 5 command burst skills.